pallets/parachain-staking/src/lib.rs

```diff
@@ -489,20 +489,23 @@ pub mod pallet {
 					selected_collators_number: collator_count,
 					total_balance: total_staked,
 				});
+				// record inactive collators
+				Self::mark_collators_as_inactive(round.current);
```

Comment:
```suggestion
				weight = weight.saturating_add(Self::mark_collators_as_inactive(round.current));
```

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3128/commits/74cbae1cc254831ba1235df4b3af97c46f965075 👍